xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v1/service/BitfinexAccountService.java

```diff
@@ -82,17 +83,26 @@ public String withdrawFunds(Currency currency, BigDecimal amount, String address
     //The wallet to withdraw from, can be “trading”, “exchange”, or “deposit”.
     String walletSelected = "exchange";
     //We have to convert XChange currencies to Bitfinex currencies: can be “bitcoin”, “litecoin” or “ether” or “tether” or “wire”.
-    return withdraw(type, walletSelected, amount, address, tag);
+    return withdraw(type, walletSelected, amount, address, tagOrPaymentId);
   }
 
   @Override
   public String withdrawFunds(WithdrawFundsParams params) throws IOException {
     if (params instanceof RippleWithdrawFundsParams) {
-      RippleWithdrawFundsParams rippleWithdrawFundsParams = (RippleWithdrawFundsParams) params;
-      return withdrawFunds(rippleWithdrawFundsParams.currency, rippleWithdrawFundsParams.amount, rippleWithdrawFundsParams.address,
-          rippleWithdrawFundsParams.tag);
-    }
-    if (params instanceof DefaultWithdrawFundsParams) {
+      RippleWithdrawFundsParams xrpParams = (RippleWithdrawFundsParams) params;
+      return withdrawFunds(
+          xrpParams.currency,
```

Comment:
using getters and setters would be preferable to be consistent with the rest of the lib.

Reply:
Done